brodan.biz/index.html

```diff
@@ -7,7 +7,7 @@
   <meta name="description" content="The personal website of Christopher 'Brodan' Hranj.">
   <title>Brodan.biz - Personal Site of Chris Hranj</title>
 
-  <link rel="shortcut icon" href="/static/images/favicon.ico" type="image/x-icon">
```

Comment:
Don't use absolute URLs it fucks up when you change the directory you serve from.